lib/integration/nation_builder.rb

```diff
@@ -3,10 +3,14 @@ class Integration::NationBuilder
   STANDARD_HEADERS = {'Accept' => 'application/json', 'Content-Type' => 'application/json'}
   ENDPOINTS = {
     list_count_page: '/supporter_counts_for_website',
+    event:           '/api/v1/sites/mayday/pages/events/%s',
+    events:          '/api/v1/sites/mayday/pages/events',
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.